### PR TITLE
React app initialization and removal of some files

### PR DESCRIPTION
**Pull request by Akanksha987**
->Initialization of react app
->Deletion of some unwanted files

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "client",
```
Comment (PatelNeelMahesh) [inline client/package.json:2]: Add your Application name here.

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (PatelNeelMahesh) [inline client/README.md:1]: Remove the default generated readme details by create-react-app, add details in readme about the components you are going to create, add about installation process.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Do the required changes.

File: client/README.md (hunk 2 above)
Comment (Akanksha987) [reply client/README.md:1]: Sure, I will do it in next PR.

**Pull request merged**